app/controllers/grant_creator_requests/review_controller.rb

```diff
@@ -7,7 +7,7 @@ def show; end
     def update
       @grant_creator_request.reviewer = current_user
       if @grant_creator_request.update(grant_creator_request_review_params)
-        send_notification unless @grant_creator_request.status_pending?
+        send_review_notification unless @grant_creator_request.status_pending?
```

Comment:
those request names could be more specific -- e.g. 'requester_review_notification' and 'admin_request_notification'